File: Coding/coding modul 5/PRAK502-2410817110007-MUHAMMAD-HAMKA-AKBARI.c
#include <stdio.h>
#include <math.h>
#include <stdlib.h>

// Function to calculate differences
void hitung(int a, int b, int c, int d, int *hasil1, int *hasil2) {
    *hasil1 = a - c; // Calculate the difference between a and c
    *hasil2 = b - d; // Calculate the difference between b and d
}

// Function to calculate the absolute sum
int mutlak(int hasil1, int hasil2) {
    return abs(hasil1) + abs(hasil2); // Return the sum of absolute values
}

int main() {
    int a, b, c, d;
    int hasil1, hasil2;

    // Input values for a, b, c, d
    scanf("%d", &a);
    scanf("%d", &b);
    scanf("%d", &c);
    scanf("%d", &d);

    // Calculate differences
    hitung(a, b, c, d, &hasil1, &hasil2);

    // Print the absolute sum
    printf("Nilai mutlak adalah: %d\n", mutlak(hasil1, hasil2));

    return 0;
}

File: coding modul 5/PRAK502-2410817110007-MUHAMMAD-HAMKA-AKBARI.py
import math

def hitung(a, b, c, d):
    hasil1 = a - c
    hasil2 = b - d
    return hasil1, hasil2

def mutlak(hasil1, hasil2):
    nilai_mutlak = abs(hasil1) + abs(hasil2)
    return nilai_mutlak

# Input nilai a, b, c, d
a, b, c, d = map(int, input().split())

# Hitung hasil1 dan hasil2
hasil1, hasil2 = hitung(a, b, c, d)

# Cetak nilai mutlak
print("Nilai mutlak adalah:", mutlak(hasil1, hasil2))

